deploy/alerts/alerts.yaml

```diff
@@ -0,0 +1,136 @@
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    prometheus: example
```

Comment:
Our default Prometheus uses `prometheus: '{{ meta.name }}'` which means this should be `prometheus: stf-default` by default for compatibility.